style-guide/components/01-components/utilities/spacing/spacing.config.json

```diff
@@ -21,6 +21,13 @@
       "context": {
         "modifier": "standard-gap--bottom"
       }
+    },
+    {
+      "name": "Variable gap - mobile tablet desktop",
+      "notes": "For differing gaps between devices",
+      "context": {
+        "modifier": "margin-top-large sm:margin-top-standard md:margin-top-none"
```

Comment:
For consistency with standard-gap classes I think we should use "--" to separate the variants (e.g. margin-top--none, sm:margin-top--standard, md:margin-top--small, lg:margin-top--large)